README.md

```diff
@@ -41,11 +41,19 @@ Add this dependency to your project's POM:
 <dependency>
   <groupId>dev.merge.hris</groupId>
   <artifactId>merge-hris-client</artifactId>
-  <version>1.1.0</version>
+  <version>1.2.2</version>
   <scope>compile</scope>
 </dependency>
 ```
 
+### Gradle users
+
+Add this dependency to your project's build file:
+
+```groovy
+compile "dev.merge.hris:merge-hris-client:1.2.2"
+```
+
```

Comment:
^should probably remove this?

Reply:
I don't think we should, we may compile this java package with `maven` but `gradle` is a very popular alternative build tool that many people consider superior (it is newer and has some extra features). It is backwards compatible, so our pom.xml should still work with it. I don't see the harm?